allennlp/tests/fixtures/bidirectional_lm/experiment.jsonnet

```diff
@@ -24,6 +24,8 @@
   },
   "model": {
     "type": "bidirectional-language-model",
+    "num_samples": 10,
+    "sparse_embeddings": true,
```

Comment:
There is a good opportunity to de-duplicate using jsonnet here. Something like the following appears to work.

```
local config = import "experiment_unsampled.jsonnet";

config + {
  "model": config["model"] + {
    "num_samples": 10,
    "sparse_embeddings": true
  }
}
```

Reply:
done